Docs: clarify the description of actions_allow_post_url (#95995)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

### actions_allow_post_url

> Sets API paths to be accessible between plugins using the POST verb. Leaving this value empty only allows remote requests to pass through the proxy, whereas setting this value lets you also send authenticated POST requests to the local server. You typically use this to enable backend communication between plugins.

Reviewer: ```suggestion
Sets API paths to be accessible between plugins using the POST verb. If the value is empty, you can only pass remote requests through the proxy. If the value is set, you can also send authenticated POST requests to the local server. You typically use this to enable backend communication between plugins.
```

> This is a comma separated list, and uses glob matching.

Reviewer: ```suggestion
This is a comma-separated list which uses glob matching.
```
